### PR TITLE
net: wifi_mgmt: Updated TWT setup response status

### PR DESCRIPTION
**Pull request by ajayparida**
Updated TWT setup response status. Removed un-necessary error strings, enum and values updated as per f/w.
@krish2718 @rado17

### CODE REVIEW

**Comment by rlubos** [comment]
@krish2718 Please review.

File: subsys/net/l2/wifi/wifi_shell.c [hunk 1]
```diff
@@ -168,6 +165,8 @@ static void handle_wifi_twt_event(struct net_mgmt_event_callback *cb)
 			      resp->setup.twt_wake_interval_ms,
 			      resp->setup.twt_interval_ms);
 		}
+	} else {
+		print(context.sh, SHELL_NORMAL, "TWT response not received\n");
```
Comment (krish2718) [inline subsys/net/l2/wifi/wifi_shell.c:169]: Should we also print the timeout for response?

**Comment by rlubos** [comment]
@krish2718 Can you revisit?

**Comment by ajayparida** [comment]
@sachinthegreen

**Review by krish2718 — APPROVED** [review]
Sorry for late review.

**Review by rlubos — APPROVED** [review]

**Comment by ajayparida** [comment]
@jukkar

**Review by jukkar — APPROVED** [review]

**Pull request merged**